### PR TITLE
Soar goodman red (600_SYZY grating, RED mode)

### PR DESCRIPTION
**Pull request by rcooke-ast**
Added support for another SOAR grating... Here is the QA output:

image

### CODE REVIEW

File: pypeit/spectrographs/soar_goodman.py [hunk 1]
```diff
@@ -335,9 +335,15 @@ def config_specific_par(self, scifile, inp_par=None):
         par = super().config_specific_par(scifile, inp_par=inp_par)
 
         # Wavelength calibrations
+        # Here is a useful website with an arc atlas
+        # http://soartelescope.org/soar/content/goodman-comparison-lamps
         if self.get_meta_value(scifile, 'dispname') == '400_SYZY':
             par['calibrations']['wavelengths']['reid_arxiv'] = 'soar_goodman_red_400_SYZY.fits'
             par['calibrations']['wavelengths']['method'] = 'full_template'
+        elif self.get_meta_value(scifile, 'dispname') == '600_SYZY_OLD':
```
Comment (profxj) [inline pypeit/spectrographs/soar_goodman.py:343]: is it really called `600_SYZY_OLD`  ?  :)

**Review by profxj — APPROVED** [review]
Am approving.  Please add tests in the DevSuite

File: pypeit/spectrographs/soar_goodman.py (hunk 1 above)
Comment (rcooke-ast) [reply pypeit/spectrographs/soar_goodman.py:343]: It is!  :-D

**Comment by rcooke-ast** [comment]
Tests added to devsuite - running full suite again now... Here is a link to the corresponding PR:

[Dev suite PR](https://github.com/pypeit/PypeIt-development-suite/pull/239)

**Review by kbwestfall — APPROVED** [review]
Looks good.  Please merge in `release` before merging with `develop` (so that we can bring `develop` up-to-date!).

**Comment by rcooke-ast** [comment]
Tests pass... merging release, and then to develop...

image

**Pull request merged**